File: VB/Person.vb
Imports Microsoft.VisualBasic
Imports System
Imports DevExpress.Xpo

Namespace B190497
	Public Class Person
		Inherits XPObject
		Public Sub New(ByVal session As Session)
			MyBase.New(session)
		End Sub

		Private fName As String
		Public Property Name() As String
			Get
				Return fName
			End Get
			Set(ByVal value As String)
				SetPropertyValue(Of String)("Name", fName, value)
			End Set
		End Property
	End Class
End Namespace

File: CS/Person.cs
using DevExpress.Xpo;

namespace B190497
{
    public class Person : XPObject
    {
        public Person(Session session) : base(session) { }

        string fName;
        public string Name
        {
            get { return fName; }
            set { SetPropertyValue<string>("Name", ref fName, value); }
        }
    }
}